plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -233,4 +233,29 @@ private static function init() {
 		}
 		return static::$mantis_markdown;
 	}
+
+	/**
+	 * Fix line indent
```

Comment:
It seems a bug from the source itself. http://parsedown.org/demo. But let me check in the other extentions. https://github.com/erusev/parsedown/wiki/Extensions-and-Related-Libraries